Packs/Legacy/Integrations/integration-Netskope.yml

```diff
@@ -293,13 +295,18 @@ script:
                 incident.name = item.alert_type + " - " + item.alert_name;
                 incident.rawJSON = JSON.stringify(item);
                 incidents.push(incident);
-                last_run = item.timestamp;
-                if (incidents >= Math.min(50, params.maxFetch)) {
+                if (latestIncidedentTimeInt == null || parseInt(item.timestamp) > latestIncidedentTimeInt) {
```

Comment:
wouldn't it cause problems if latestIncidedentTimeInt is null?

Reply:
If it is null the first condition will be met and the second one won't even be checked